README.md

```diff
@@ -313,6 +313,7 @@ We also ship a config for the popular [pull app](https://github.com/apps/pull) i
 
 ## Join The Community
 
-You can find us on the [Universal Blue Discord](https://discord.gg/f8MUghG5PB) and also discuss Bazzite on our [Discourse forums](https://universal-blue.discourse.group/c/bazzite/5).  
```

Comment:
Small nitpick - I'm not sure about removing references to Universal Blue here since Bazzite uses Ublue's Discord/Discourse

Reply:
I'm going to actually going to mark this as a draft anyways, and redo some of it.